src/ResourceManager/Websites/Commands.Websites/Models.Websites/WebsiteBaseNotMandatoryCmdlet.cs

```diff
@@ -0,0 +1,32 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Websites.Models;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.Websites.Models.Websites;
+
+namespace Microsoft.Azure.Commands.Websites
+{
+    public class WebsiteBaseNotMandatoryCmdlet : WebsitesBaseClient
```

Comment:
not really related to this change but I think that `WebsitesBaseClient` should be renamed as it's misleading, it is actually a Cmdlet and not a Client (only holds a client).